js/_switchContent.js

```diff
@@ -0,0 +1,23 @@
+export default function changePages() {
+  const buttons = document.querySelectorAll("[data-js=page__button]");
+  const mains = document.querySelectorAll("[data-js=main]");
+  const header = document.querySelector("h1");
+
+  buttons.forEach((button) => {
+    button.addEventListener("click", () => {
+      const activeButtons = document.querySelector("button.active");
+      activeButtons.classList.remove("active");
```

Comment:
I would recommend to use singular as variable names if you only store one value and plural if you store multiple (f.e. an array). In this case, `activeButtons` is a reference to only one active button and not multiple buttons. For better readability:
```suggestion
      const activeButton = document.querySelector("button.active");
      activeButton.classList.remove("active");
```